core-processor/src/configs.rs

```diff
@@ -104,6 +108,8 @@ pub struct BlockConfig {
     pub outgoing_limit: u32,
     /// Host function weights.
     pub host_fn_weights: HostFnWeights,
+    /// Static host function weights.
+    pub static_host_fn_weights: StaticHostFnWeights,
```

Comment:
Why static? Why host fn?  Rename it everywhere. Our runtime use sandbox and don't care whether it use host or embedded executor (idiomatically). Actually I don't know what else do you wanna to put inside `static_host_fn_weights`, but can propose you: `executor_backend_weights` or `sandbox_weights`.

Reply:
Agree